### DISCUSSION TITLE
Git clone error message

### DISCUSSION THREAD

**Discussion by Buchiken11** [community/community#58343] [New to GitHub]
fatal: unable to access 'https://my tokengithub.com/Buchiken11/alx-low_level_programming.git/': Could not resolve host: ghp_04H7h

**Comment by airtower-luna** [comment]
That looks like you tried to insert the token into the repository URL in an incorrect way, and the token (or part of it) was interpreted as the hostname of the URL.

You should not include the token in the URL at all. If you don't want to have to provide it for every Git operation, enable [Git Credential Storage](https://git-scm.com/book/en/v2/Git-Tools-Credential-Storage). Usually the `cache` helper is good, combined with a password manager to store the token long-term.
